opencti-platform/opencti-graphql/src/modules/case/case-incident/case-incident-types.ts

```diff
@@ -0,0 +1,35 @@
+import type { BasicStoreEntity, StoreEntity } from '../../../types/store';
+import type { StixDomainObject, StixOpenctiExtensionSDO } from '../../../types/stix-common';
+import { STIX_EXT_OCTI } from '../../../types/stix-extensions';
+
+export const ENTITY_TYPE_CONTAINER_CASE_INCIDENT = 'Case-Incident';
+
+export interface BasicStoreEntityCaseIncident extends BasicStoreEntity {
+  name: string,
+  description: string,
+  context: string,
+  object_refs: Array<string>,
+}
+
+export interface StoreEntityCaseIncident extends StoreEntity {
+  name: string,
+  type: string,
+  severity: string,
+  priority: string,
+  description: string,
+  object_refs: Array<string>,
+  incidentResponseType: string,
+}
+
+export interface StixCaseIncident extends StixDomainObject {
+  name: string,
+  description: string,
+  severity: string,
+  priority: string,
+  confidence: number,
```

Comment:
It's already handle by the StixDomainObject class (same for feedback)

Reply:
✅